third_party/Build.mk

```diff
@@ -70,5 +70,7 @@ third_party/localtests: cargo_version_check sandbox_setup
 
 .PHONY: build/cargo-host/release/elf2tab
 build/cargo-host/release/elf2tab: cargo_version_check sandbox_setup
-	cd third_party/elf2tab && \
+	cp -rp -t build third_party/elf2tab && \
+	rm -f build/elf2tab/Cargo.lock && \
```

Comment:
Do we need to remove this? Shouldn't we remove it from `sandbox_setup` and `BWRAP` instead?

Reply:
in your sandbox_setup you made an empty file, which I would guess you probably needed because bwrap needs an actual file to map into the sandbox.   But simply removing the Cargo.lock file in third_party/elf2tab made the build work (except that the local git repository was now dirty).    If we remove it from sandbox_setup I expect bwrap will no longer solve the problem.    Oh, with this change I should clean up at least that part of sandbox_setup and remove the mapping from the bwrap command line?   Is that what you're suggesting?

Reply:
Yes, I am suggesting you remove it from `sandbox_setup` as well as the `bwrap` bind mount.

Reply:
I cleaned up the build/Cargo.lock/elf2tab from the main Makefile now.    Leaving the BWRAP definition in there for now, still useful to make sure builds we do by hand don't depend on the network, and the use of BWRAP keeps builds we do by hand from sprinkling new Cargo.lock files in our source tree.